### PR TITLE
Bye bye Google Plus

### PR DESCRIPTION
**Pull request by escopecz**
Removing all mentions of Google+ as this service does not exist anymore

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | N
| Automated tests included? | /
| Related user documentation PR URL | /
| Related developer documentation PR URL | /
| Issues addressed (#s or URLs) | /
| BC breaks? | Y
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Removing all mentions of Google+ as this social network does not exist anymore. It's time to get rid of it.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Make for example Twitter plugin public. No need to authenticate, but feel free to do so.
3. Go to a form (or create a new one) with the "Social Login" field.
4. Go to the form preview and click the Twitter button.

It should load a site from Twitter saying that the token is wrong. And they are right unless you configured it properly. But the error message is a successful test here. There were some errors fixed to get to this message.

#### List backwards compatibility breaks:
1. See the Upgrading doc changes.

### CODE REVIEW

File: app/bundles/PluginBundle/Form/Type/FieldsTypeTrait.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ function (FormEvent $event) use ($options, $integrationFields, $mauticFields, $f
 
                 foreach ($mauticFields as $key => $value) {
                     if (is_array($mauticFields)) {
-                        $mauticFields[$key] = array_flip($value);
+                        $mauticFields[$key] = $value;
```
Comment (lukassykora) [inline app/bundles/PluginBundle/Form/Type/FieldsTypeTrait.php]: Array flip needs to stay:
`$mauticFields[$key] = array_flip($value);`

File: app/bundles/FormBundle/Controller/FieldController.php [hunk 2]
```diff
@@ -155,6 +160,7 @@ public function newAction()
                     'field'         => $formField,
                     'id'            => $keyId,
                     'formId'        => $formId,
+                    'formName'      => $formEntity->generateFormName(),
```
Comment (lukassykora) [inline app/bundles/FormBundle/Controller/FieldController.php]: If new form $formEntity is null.
`'formName'      => $formEntity !== null ? $formEntity->generateFormName() : ""`

Comment (escopecz) [reply app/bundles/FormBundle/Controller/FieldController.php]: The string cannot be empty. I got route errors if it were and that's why I added the formName here.

Comment (lukassykora) [reply app/bundles/FormBundle/Controller/FieldController.php]: The problem is that $formEntity is null if you create a new form.

Comment (escopecz) [reply app/bundles/FormBundle/Controller/FieldController.php]: Good catch. Thanks!

**Pull request merged**